jwst/pipeline/calwebb_detector1.py

```diff
@@ -134,8 +134,12 @@ def process(self, input):
             input, ints_model = self.ramp_fit(input)
 
         # apply the gain_scale step to the exposure-level product
-        self.gain_scale.suffix = 'gain_scale'
-        input = self.gain_scale(input)
+        # XXX JP-2651
+        if input is not None:
+            self.gain_scale.suffix = 'gain_scale'
+            input = self.gain_scale(input)
+        else:
+            log.info("NoneType returned from ramp_fit")
```

Comment:
The message could be expanded just a bit here to also mention that the `gain_scale` step will be skipped.